universal-application-tool-0.0.1/app/services/program/ProgramDefinition.java

```diff
@@ -103,6 +107,29 @@ public String getLocalizedDescription(Locale locale) throws TranslationNotFoundE
     }
   }
 
+  public ImmutableSet<Locale> getSupportedLocales() {
```

Comment:
(blocking) this should have a docstring describing what it means to be "supported"

Reply:
Thank you for catching that! Added